cmd/mdatagen/internal/sampleconnector/internal/metadata/generated_config_test.go

```diff
@@ -27,11 +27,42 @@ func TestMetricsBuilderConfig(t *testing.T) {
 			name: "all_set",
 			want: MetricsBuilderConfig{
 				Metrics: MetricsConfig{
-					DefaultMetric:            MetricConfig{Enabled: true},
-					DefaultMetricToBeRemoved: MetricConfig{Enabled: true},
-					MetricInputType:          MetricConfig{Enabled: true},
-					OptionalMetric:           MetricConfig{Enabled: true},
-					OptionalMetricEmptyUnit:  MetricConfig{Enabled: true},
+					DefaultMetric: MetricConfig{
+						Enabled:             true,
+						AggregationStrategy: AggregationStrategySum},
+					DefaultMetricToBeRemoved: MetricConfig{
+						Enabled:             true,
+						AggregationStrategy: AggregationStrategySum},
+					MetricInputType: MetricConfig{
+						Enabled:             true,
+						AggregationStrategy: AggregationStrategySum},
```

Comment:
Let's keep the format consistent
```suggestion
						Enabled:             true,
						AggregationStrategy: AggregationStrategySum,
					},
```